src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ConsentOrderApprovedController.java

```diff
@@ -81,7 +82,12 @@
             log.info("isApprovedConsentOrderNotificationLetterEnabled toggle set to: {}",
                 featureToggleService.isApprovedConsentOrderNotificationLetterEnabled());
 
-            if (featureToggleService.isApprovedConsentOrderNotificationLetterEnabled()) {
+            log.info("isPaperApplication set to: {}", isPaperApplication(caseData));
+
+            if (featureToggleService.isApprovedConsentOrderNotificationLetterEnabled() && isPaperApplication(caseData)) {
```

Comment:
I think this might be more easily read/understood with two if statements:
- You can move the builder (line 97) to before either if statement
- The outer if will be the feature-toggle if. It will go all the way to what's line 104.
- the inner if will be only to check whether it's paper case or not. It will only encompass the lines until what's now line 94.

Reply:
Yep totally agree - great spot, changes made 👍